src/app.rs

```diff
@@ -88,11 +96,20 @@ impl Application for AppModel {
                     Err((_errors, config)) => config,
                 })
                 .unwrap_or_default(),
+            dmidecode: None,
             lscpu: None,
             lspci: None,
             lsusb: None,
         };
 
+        let dmidecode_cmd = std::process::Command::new("dmidecode -t baseboard").output();
```

Comment:
You probably need to run this as sudo for it to work.

Reply:
The issue with that is that there will not be a prompt to ask the user for that currently. It looks like this gives some info without sudo:

cat /sys/devices/virtual/dmi/id/board_{vendor,name,version}